resources/assets/lib/stores/channel-store.ts

```diff
@@ -188,4 +192,62 @@ export default class ChannelStore {
 
     this.loaded = true;
   }
+
+  @action
+  private handleChatChannelNewMessages(channelId: number, json: MessageJSON[]) {
+    const messages = json.map((messageJson) => {
+      if (messageJson.sender != null) this.userStore.getOrCreate(messageJson.sender_id, messageJson.sender);
```

Comment:
(probably not this line but just so it can be marked resolved)

User info is missing on initial load

![](https://s.myconan.net/2020-10/firefox_2020-10-16_20-02-09.png)

Reply:
this change might have been dependent on changes to channel loading 🤔